docs/gettingStarted.md

```diff
@@ -0,0 +1,63 @@
+### Step 1 - Prerequisites
+
+Add [redux-thunk](https://github.com/gaearon/redux-thunk) middleware to your store,
+ following instructions at that repo.
+
+### Step 2 - [Config](./inputsConfig.md)
+
+Both the reducer and input props creator take a inputsConfig object that defines how inputs are initialized and validated.
+
+### Example
+
+    const inputsConfig = {
+        homePrice: {
+            defaultValue: 300000,
+            validator: (value) => value > 0 && value < 350000000
+        }
+    }
+
+Create an input configuration object and add a redux-inputs reducer to your redux store in your project's reducers file.
```

Comment:
> ...and add a redux-inputs reducer to your redux store...

This is not shown in the Step 2 example at all. This should probably be part of Step 3.